logs/log-20200318T222651.txt

```diff
@@ -0,0 +1,2640 @@
+555  [main] WARN  com.zaxxer.hikari.HikariConfig - database.postgres_dc.db - using dataSourceClassName and ignoring jdbcUrl.
```

Comment:
I think the logs should be removed from PR

Reply:
Done

Reply:
@Dmitrii-Shevchenko  I still see this files in list of changes in PR.

Reply:
Fixed, thank you!